xml/ssh_banners.xml

```diff
@@ -381,9 +392,10 @@ fingerprint SSH servers.
     <param pos="0" name="os.product" value="Linux"/>
     <param pos="0" name="os.version" value="8.0"/>
   </fingerprint>
-  <fingerprint pattern="^OpenSSH_([^\s]+)\s+(Debian.+squeeze.*)$">
+  <fingerprint pattern="^OpenSSH_([^\s]+)\s+(Debian.+[-6|squeeze].*)$">
```

Comment:
Do you have an example of this banner w/o `Debian-6` in it?  It would seem that the `-6` shouldn't be optional.  Also, if you test this on Rubular.com you will find that the regex matches `OpenSSH_4.1p1 Debian-7ubuntu4`.

I recommend something similar to 
`^OpenSSH_([^\s]+)\s+(Debian-6(?:squeeze)?.*)$`